bcda/client/bluebutton.go

```diff
@@ -190,48 +186,38 @@ func (bbc *BlueButtonClient) GetClaim(jobData models.JobEnqueueArgs, mbi string,
 	header := make(http.Header)
 	header.Add("IncludeTaxNumbers", "true")
 
-	mbiHash, err := HashIdentifier(mbi)
-	if err != nil {
-		return nil, err
-	}
-
 	params := GetDefaultParams()
-	params.Set("mbi", mbiHash)
 	params.Set("excludeSAMHSA", "true")
 
 	updateParamWithServiceDate(&params, claimsWindow)
 	updateParamWithLastUpdated(&params, jobData.Since, jobData.TransactionTime)
 
-	u, err := bbc.getURL("Claim", params)
+	u, err := bbc.getURL("Claim/_search", params)
```

Comment:
We use getURL above as well.  Some nitpicks/thoughts:
- We should return the same thing each time (we return empty string above and nil here)
- Do we want to return anything more useful than an empty string?  Looking at getURL it is already catching an error, maybe getURL doesnt need to return an error?